pages/common/ed.md

```diff
@@ -1,21 +1,22 @@
 # ed
 
 > The original Unix text editor.
+> See also: `awk`, `sed`.
 > More information: <https://manned.org/ed.1>.
 
-- Start an interactive editor session:
+- Start an interactive editor session with an empty document:
 
 `ed`
 
-- Start an interactive editor session with the specified prompt:
+- Start an interactive editor session with an empty document and the specified prompt:
 
-`ed --prompt='> '`
+`ed --prompt="> "`
```

Comment:
Single quotes are preferable here to avoid a `$` for example being interpreted by the shell.